code/modules/roguetown/roguemachine/ATM.dm

```diff
@@ -74,8 +74,10 @@
 				SStreasury.generate_money_account(P.get_real_price(), H)
 				if(!(H.job in GLOB.noble_positions) && !HAS_TRAIT(H, TRAIT_NOBLE))
 					var/T = round(P.get_real_price() * SStreasury.tax_value)
-					if(T != 0)
+					if(T > 1)
 						say("Your deposit was taxed [T] mammon.")
+					else
+						say("Your deposit was taxed 1 mammon.")
```

Comment:
You don't need this if else thing if both lines are the same thing- If T is one it was saying "Your deposit was taxed [T] mammon." i. e. this change seems unnecessary and redundant?